xml_inscriptions/bib_only/MO.Col.UM.MAA.L.59.2.xml

```diff
@@ -28,14 +28,42 @@
                      <p/>
```

Comment:
Make sure to move to correct folder, if you haven't already (it doesn't seem like it has here).